lua/neogit/lib/popup.lua

```diff
@@ -575,6 +575,21 @@ function M:show()
       }
     end,
   }
+
+  if buf_options.kind == "floating" then
```

Comment:
I'd just keep the constructor as is, and instead of conditionally adding the autocmd, put the condition in the callback function. That'll keep the constructor consistent with the other callers.

Reply:
Done in the latest force push.